keepalived/vrrp/vrrp.c

```diff
@@ -2562,7 +2564,7 @@ reset_vrrp_state(vrrp_t *old_vrrp, vrrp_t *vrrp)
 	vrrp->init_state = old_vrrp->state;
 	vrrp->wantstate = old_vrrp->state;
 	if (!old_vrrp->sync)
-		vrrp->effective_priority = old_vrrp->effective_priority;
+		vrrp_set_effective_priority(vrrp, old_vrrp->effective_priority);
```

Comment:
It's there for the purpose of being very specific what the state us after reload. But it may be obsoleted by the previous suggestion to specifically log changes.